routes/engine.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../lib/engine_list'
+require_relative '../lib/engine'
+require_relative '../lib/error'
+
+namespace '/engine' do
+  get '' do
+    code = 200
+    response = EngineList.new
+
+    respond_to do |f|
+      f.json { [code, response.bamboo_json] }
+      f.xml { [code, response.bamboo_xml] }
+    end
+  end
+
+  get '/:engine' do
+    if !valid_engine?
+      code = 404
+      response = Error.new('unknown engine')
+    else
+      code = 200
```

Comment:
I've replicated the functionality in Morphsvc. But I'm confused about why the `/engine` endpoints return a `200 ok` while the `/analysis` endpoints return `201 created`.

Reply:
The 201 response was a continuation of the concept that what the morphology service was doing was *creating* a new Annotation resource for each request, as opposed to returning a static document.  It's a legacy of the infrastructure it was meant to be a part of. Under the current implementation, it probably makes more sense for it to return 200 for the /analysis endpoint.